lib_nbgl/include/nbgl_use_case_v2.h

```diff
@@ -0,0 +1,235 @@
+                                 const char                   *tagline,
+                                 const uint8_t                *initSettingPage, // Set to NULL if settings shouldn't be displayed right away
+                                 const nbgl_contentInfoList_t *infosList,
+                                 const nbgl_genericContents_t *settingContents,
+                                 nbgl_callback_t               quitCallback);
+
+void nbgl_useCaseHomeExtAndSettings(const char                   *appName,
+                                    const nbgl_icon_details_t    *appIcon,
+                                    const char                   *tagline,
+                                    const uint8_t                *initSettingPage, // Set to NULL if settings shouldn't be displayed right away
+                                    const nbgl_contentInfoList_t *infosList,
+                                    const nbgl_genericContents_t *settingContents,
+                                    nbgl_callback_t               quitCallback,
+                                    const char                   *actionText, // Set to NULL if no additional action
+                                    nbgl_callback_t               actionCallback); // Set to NULL if no additional action
+
+// Note no specific support for plugin right now, this is so specific it should probably be handled in app side using nbgl_useCaseGenericNav?
+
+/*
+ * TX review, exist in three variants "transaction", "message" and "operation"
+ * Then all wording are handled by the SDK and not exposed to the user.
+ * This bundles nbgl_useCaseReviewStart / nbgl_useCaseStaticReview and nbgl_useCaseConfirm
+ */
+
+// Note, maybe at some point an additional API could be proposed to take a nbgl_genericContents_t instead of a tagValueList
+// Could be an *Ext version..
+
+void nbgl_useCaseTransactionReview(
+    const nbgl_layoutTagValueList_t *tagValueList,
+    const nbgl_icon_details_t *icon,
+    const char *reviewTitle,
+    const char *reviewSubTitle, /* Most often this is empty, but sometimes indicates a path / index */
+    const char *finish_page_text, /* unused on Nano */
+    nbgl_choiceCallback_t choice_callback);
+
+void nbgl_useCaseMessageReview(
+    const nbgl_layoutTagValueList_t *tagValueList,
+    const nbgl_icon_details_t *icon,
+    const char *reviewTitle,
+    const char *reviewSubTitle, /* Most often this is empty, but sometimes indicates a path / index */
+    const char *finish_page_text, /* unused on Nano */
+    nbgl_choiceCallback_t choice_callback);
+
+void nbgl_useCaseOperationReview(
+    const nbgl_layoutTagValueList_t *tagValueList,
+    const nbgl_icon_details_t *icon,
+    const char *reviewTitle,
+    const char *reviewSubTitle, /* Most often this is empty, but sometimes indicates a path / index */
+    const char *finish_page_text, /* unused on Nano */
+    nbgl_choiceCallback_t choice_callback);
+
```

Comment:
Do we really need 3 distinct APIs with the same parameters list?
Could we simplify, just using an additional parameter (I don't know the internal functions details.....)?